example/lib/main.dart

```diff
@@ -7,7 +7,7 @@ void main() {
     activateLicense();
 
     final status = LexActivator.IsLicenseGenuine();
-    LexActivator.SetCallback(callback: (status) {
+    LexActivator.SetLicenseCallback(callback: (status) {
```

Comment:
@mumin-khan SetLicenseCallback() should be registered before making call to IsLicenseGenuine()

Reply:
@ahmad-kemsan done.